docs/_documentations/image-registry-credentials.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: docs
+title: Image registry guidance for Eclipse Codewind
+description: Image registry guidance for Eclipse Codewind
+keywords: users, projects, Kubernetes, image, registry, deployment, push
+duration: 5 minutes
+permalink: dockerregistry
+type: document
+parent: installoncloud
+order: 1
+---
+# Codewind Docker registry guidance
+When you run Codewind on Kubernetes, Codewind uses [`buildah`](https://github.com/containers/buildah) to build container images from Dockerfiles and to push the images to an image registry that you specify. You can use the `Image Registry Manager` in your Codewind IDE to configure the registry to use. 
+
+Codewind is flexible with the registry that can be used as long as `buildah` and the node that Codewind is running on can resolve the registry hostname. If `buildah` is unable to resolve the registry hostname,`buildah` is unable to push your projects to the registry, and deployment on Kubernetes is prevented.
+
+Before you proceed, see the [examples](#examples) section on recommended values for common registries:
+
+## Examples:
+The following deployment registries have been tested and verified with Codewind:
+- Dockerhub:
+    - Host: `docker.io`
```

Comment:
These keys have also changed. The new Codewind UI has no such fields.

They are address, username, password and namespace.

Another question is, should we provide a screenshot of the Image Registries UI with this doc? It'll make things easier for anyone reading it.